### ISSUE TITLE
Types from `lib/compiler/src/ml_parser/parser.d.ts` cannot be resolved

### ISSUE THREAD

**Issue by nicojs** [prettier/angular-html-parser#42]
### Which @angular/* package(s) are the source of the bug?

Don't known / other

### Is this a regression?

Yes, since v7.0.0.

### Description

The return type of `parse` from `lib/compiler/src/ml_parser/parser.d.ts` cannot be resolved. This is because imports there lack the `.js` postfix. This is only a problem for the `.d.ts` files, so runtime is unaffected.

### Please provide the exception or error you saw

See this example

```ts
 const parser = (await import('angular-html-parser')).parse;
  const { RecursiveVisitor, visitAll } = await import('angular-html-parser/lib/compiler/src/ml_parser/ast.js');

  const { rootNodes, errors } = parser(text, {
    canSelfClose: true,
    allowHtmComponentClosingTags: true,
    isTagNameCaseSensitive: true,
  });

  if (errors.length !== 0) {
    throw new ParseError(errors[0].msg, fileName, toSourceLocation(errors[0].span.start));
  }
```

Here, `errors[0].msg` is resolved as type `any`.

![image](https://github.com/user-attachments/assets/5d26cdd4-d2da-4fdc-8b33-7a2b7b76d29e)

### Please provide the environment you discovered this bug in (run `ng version`)

```true
I'm not using Angular, only the `angular-html-parser`
```

### Anything else?

This is a regression in v7.0.0. The `lib/compiler/src/ml_parser/parser.d.ts` file in the previous version (v6.0.2) included the correct `.js` extensions in imports. Was there a problem with the release?

**Comment by fisker** [comment]
Sorry about that, [this script](https://github.com/prettier/angular-html-parser/blob/eceda60342aed26ba95a4ca23bae5564948d3bcf/packages/angular-html-parser/package.json#L20) supposed to fix the extension but somehow it didn't work as expected.

**Comment by fisker** [comment]
Should fixed in v7.0.1. Thanks for your report.
